examples/1-execute-query-mutation/src/Previewer.re

```diff
@@ -93,7 +91,7 @@ let make = () => {
         </div>
         <button
           className={PreviewerStyles.colors.query->PreviewerStyles.button}
-          onClick={_event => executeQuery()}>
+          onClick={_event => executeQuery() |> ignore}>
```

Comment:
These ignores had to get added b/c `.subscribe` returns one of `wonka`'s `subscriptionT` types (basically just allows you to unsubscribe from the query).